community/examples/htc-htcondor.yaml

```diff
@@ -21,7 +21,9 @@ vars:
   region: us-central1
   zone: us-central1-c
   disk_size_gb: 100
-  new_image_family: htcondor-10x
+  new_image:
```

Comment:
Should this be more descriptive than just `new`?

Reply:
I've tried to not change any names / change semantics, the  `new_image` is just a derivative of `new_image_family`.